### DISCUSSION TITLE
Why the example.ipynb has more solving time than the SCIP, even SCIP has more  branching nodes?

### DISCUSSION THREAD

**Discussion by dashijia** [ds4dm/ecole#327] [Questions]
![node_time_diff](https://user-images.githubusercontent.com/48823990/165236916-c0663a70-6514-4e58-abd7-2d44ef0caf85.png)

**Comment by rezabayani** [comment]
It should be the other way around. Even in cases where GNN performs way worse than SCIP, I get a better time. Are you using GPU? And by the way, can you tell me your model parameters because I have been struggling to train the GNN to this extent.
